packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Field } from 'formik';
+import { DesktopWrapper, Dropdown, MobileWrapper, Text, SelectNative } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const TradingAssessmentDropdown = ({ item_list, onChange, values, setFieldValue, setEnableNextSection }) => {
+    React.useEffect(() => {
+        checkIfAllFieldsFilled();
```

Comment:
I don’t understand from the first glance what this function is doing. can you some how rename it? it seems, that it just enables next field. But the name is about checking if all fields have been fulfilled

Reply:
As discussed will rename it